ghost/core/core/frontend/apps/private-blogging/lib/middleware.js

```diff
@@ -55,7 +55,8 @@ const privateBlogging = {
             name: 'ghost-private',
             maxAge: constants.ONE_MONTH_MS,
             signed: false,
-            sameSite: 'none'
+            sameSite: urlUtils.isSSL(config.get('url')) ? 'none' : 'lax',
+            secure: urlUtils.isSSL(config.get('url'))
```

Comment:
Ghost is intended to work locally, over `http`, as well in production, over `https`. 

Before, the `sameSite` attribute for the cookie was always set to `none`. 

Here's what the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies) have to say about the sameSite attribute:

> None specifies that cookies are sent on both originating and cross-site requests, but only in secure contexts (i.e., if SameSite=None then the Secure attribute must also be set).

But for the local-development case in the bug report, the `secure` flag wasn't being set and *can't* be set. 

Here, the proposed solution it continue to `sameSite: none` just as before in the https/production case, but resolves the issue by setting `sameSite: lax` for the local case. 

As @joe-blocher points out, Ghost is already using the exact same solution in `core/server/services/auth/session/express-session.js` for a similar case. 

This change LGTM. I recommend merging.